packages/docs/package.json

```diff
@@ -19,8 +19,8 @@
         "format": "prettier --write ."
     },
     "dependencies": {
-        "@docusaurus/core": "^2.0.1",
-        "@docusaurus/preset-classic": "^2.0.1",
+        "@docusaurus/core": "2.0.1",
```

Comment:
There have been a few Docusaurus releases, and this particular install broke the build. I spend about an hour on getting it to work, then gave up and pinned the dependency.